project/ISSUE-TRIAGE.md

```diff
@@ -86,19 +66,25 @@ labels.
 | Level            | Experience level guideline                                                                                               |
 |------------------|--------------------------------------------------------------------------------------------------------------------------|
 | exp/beginner     | You have made less than 10 contributions in your life time to any open source project.                                   |
-| exp/novice       | You have made more than 10 contributions to an open source project or at least 5 contributions to Docker.                | 
-| exp/proficient   | You have made more than 5 contributions to Docker which amount to at least 200 code lines or 1000 documentation lines.   | 
-| exp/expert       | You have made less than 20 commits to Docker which amount to 500-1000 code lines or 1000-3000 documentation lines.       | 
-| exp/master       | You have made more than 20 commits to Docker and greater than 1000 code lines or 3000 documentation lines.               | 
+| exp/intermediate | You have made more than 5 contributions to Docker which amount to at least 200 code lines or 1000 documentation lines.   |
+| exp/expert       | You have made less than 20 commits to Docker which amount to 500-1000 code lines or 1000-3000 documentation lines.       |
 
 As the table states, these labels are meant as guidelines. You might have
 written a whole plugin for Docker in a personal project and never contributed to
 Docker. With that kind of experience, you could take on an <strong
-class="gh-label expert">exp/expert</strong> or <strong class="gh-label
-master">exp/master</strong> level task.
+class="gh-label expert">exp/expert</strong> level task.
+
+### 3. Prioritizing issue
 
-Contributors can add labels by using a `+exp/expert` format in issue comment.
+When attached to a specific milestone, an issue can be attributed one of the
+following labels to indicate their degree of priority (from more urgent to less
+urgent).
 
+| Priority    |
```

Comment:
In swarm, we also have a `P3`.

Here's how we defined our priorities:

* **P3**: Best effort. Those are nice to have / minor issues
* **P2**: Normal priority: Default priority applied
* **P1**: Important: **P1** issues are a top priority and a must-have for the next release
* **P0**: Urgent: Security, critical bugs, blocking issues. **P0** basically means *drop everything you are doing until this issue is addressed*